### DISCUSSION TITLE
Has anyone have had problems implementing the library with Angular 15?

### DISCUSSION THREAD

**Discussion by 56kgdev** [abacritt/angularx-social-login#663] [Q&A] [answered]
Hi, i´ve been trying to implement the library but I receive this errors:

`./src/app/main-providers.instances.ts:1:0-106 - Error: Module not found: Error: Package path ./providers/facebook-login-provider is not exported from package C:\BOT\SIVU\web_sivu_frontoffice\node_modules\@abacritt\angularx-social-login (see exports field in C:\BOT\SIVU\web_sivu_frontoffice\node_modules\@abacritt\angularx-social-login\package.json)

./node_modules/@abacritt/angularx-social-login/fesm2020/abacritt-angularx-social-login.mjs:40:26-30 - Error: export 'skip' (imported as 'skip') was not found in 'rxjs' (possible exports: ArgumentOutOfRangeError, AsyncSubject, BehaviorSubject, ConnectableObservable, EMPTY, EmptyError, GroupedObservable, NEVER, Notification, NotificationKind, ObjectUnsubscribedError, Observable, ReplaySubject, Scheduler, Subject, Subscriber, Subscription, TimeoutError, UnsubscriptionError, VirtualAction, VirtualTimeScheduler, animationFrame, animationFrameScheduler, asap, asapScheduler, async, asyncScheduler, bindCallback, bindNodeCallback, combineLatest, concat, config, defer, empty, forkJoin, from, fromEvent, fromEventPattern, generate, identity, iif, interval, isObservable, merge, never, noop, observable, of, onErrorResumeNext, pairs, partition, pipe, queue, queueScheduler, race, range, 
scheduled, throwError, timer, using, zip)

./node_modules/@abacritt/angularx-social-login/fesm2020/abacritt-angularx-social-login.mjs:42:27-31 - Error: export 'skip' (imported as 'skip') was not found in 'rxjs' (possible exports: ArgumentOutOfRangeError, AsyncSubject, BehaviorSubject, ConnectableObservable, EMPTY, EmptyError, GroupedObservable, NEVER, Notification, NotificationKind, ObjectUnsubscribedError, Observable, ReplaySubject, Scheduler, Subject, Subscriber, Subscription, TimeoutError, UnsubscriptionError, VirtualAction, VirtualTimeScheduler, animationFrame, animationFrameScheduler, asap, asapScheduler, async, asyncScheduler, bindCallback, bindNodeCallback, combineLatest, concat, config, defer, empty, forkJoin, from, fromEvent, fromEventPattern, generate, identity, iif, interval, isObservable, merge, never, noop, observable, of, onErrorResumeNext, pairs, partition, pipe, queue, queueScheduler, race, range, 
scheduled, throwError, timer, using, zip)

./node_modules/@abacritt/angularx-social-login/fesm2020/abacritt-angularx-social-login.mjs:61:34-40 - Error: export 'filter' (imported as 'filter') was not found in 'rxjs' (possible exports: ArgumentOutOfRangeError, AsyncSubject, BehaviorSubject, ConnectableObservable, EMPTY, EmptyError, GroupedObservable, NEVER, Notification, NotificationKind, ObjectUnsubscribedError, Observable, ReplaySubject, Scheduler, Subject, Subscriber, Subscription, TimeoutError, UnsubscriptionError, VirtualAction, VirtualTimeScheduler, animationFrame, animationFrameScheduler, asap, asapScheduler, async, asyncScheduler, bindCallback, bindNodeCallback, combineLatest, concat, config, defer, empty, forkJoin, from, fromEvent, fromEventPattern, generate, identity, iif, interval, isObservable, merge, never, noop, observable, of, onErrorResumeNext, pairs, partition, pipe, queue, queueScheduler, race, range, scheduled, throwError, timer, using, zip)

./node_modules/@abacritt/angularx-social-login/fesm2020/abacritt-angularx-social-login.mjs:117:39-43 - Error: export 'take' (imported as 'take') was not found in 'rxjs' (possible exports: ArgumentOutOfRangeError, AsyncSubject, BehaviorSubject, ConnectableObservable, EMPTY, EmptyError, GroupedObservable, NEVER, Notification, NotificationKind, ObjectUnsubscribedError, Observable, ReplaySubject, Scheduler, Subject, Subscriber, Subscription, 
TimeoutError, UnsubscriptionError, VirtualAction, VirtualTimeScheduler, animationFrame, animationFrameScheduler, asap, asapScheduler, async, asyncScheduler, bindCallback, bindNodeCallback, combineLatest, concat, config, defer, empty, forkJoin, from, fromEvent, fromEventPattern, generate, identity, iif, interval, isObservable, merge, never, noop, observable, of, onErrorResumeNext, pairs, partition, pipe, queue, queueScheduler, race, range, scheduled, throwError, timer, using, zip)

./node_modules/@abacritt/angularx-social-login/fesm2020/abacritt-angularx-social-login.mjs:405:37-41 - Error: export 'take' (imported as 'take') was not found in 'rxjs' (possible exports: ArgumentOutOfRangeError, AsyncSubject, BehaviorSubject, ConnectableObservable, EMPTY, EmptyError, GroupedObservable, NEVER, Notification, NotificationKind, ObjectUnsubscribedError, Observable, ReplaySubject, Scheduler, Subject, Subscriber, Subscription, 
TimeoutError, UnsubscriptionError, VirtualAction, VirtualTimeScheduler, animationFrame, animationFrameScheduler, asap, asapScheduler, async, asyncScheduler, bindCallback, bindNodeCallback, combineLatest, concat, config, defer, empty, forkJoin, from, fromEvent, fromEventPattern, generate, identity, iif, interval, isObservable, merge, never, noop, observable, of, onErrorResumeNext, pairs, partition, pipe, queue, queueScheduler, race, range, scheduled, throwError, timer, using, zip)`

Most of the error are about rxjs methods, I´ve tried changing the imports for each one and is not solving it.
then reading the documentation I see that is listed only to the Angular 13,14 versions.

Has anyone faced an issue installing it on Angular 15? Or am I doing something wrong?

**Comment by enostavnik** [comment]
I'm using it on angular 15. I used the npm i -f option to force the install. Maybe try this.

**Reply by 56kgdev** [reply]
Didn´t work :/ 
still getting the same error related to the 'rxjs' methods.

I am trying to run my project with minimal configuration, just adding the Module and Providers in the app.module, am I missing something?

**Comment by akakstetter** [comment]
Working for me too on Angular 15, but i faced the issue while upgrading my app. What version of 'rxjs' are you using? I had to upgrade it as well (to 7.5.5).

**Comment by Heatmanofurioso** [comment, marked as answer]
We're keeping a rather recent version of RXJS in support with the lib to fight deprecation.
The latest version uses 7.8.0

**Reply by 56kgdev** [reply]
Thanks a lot! I have tried everything and last week I created another branch of my project upgrading rxjs to 7.8.0 and everything works fine so far!
